src/main/webapp/app/quiz/participate/quiz.component.ts

```diff
@@ -70,6 +70,7 @@ export class QuizComponent implements OnInit, OnDestroy {
     questionScores = {};
     quizId: number;
     interval: any;
+    arrayOfAnswer = new Array<any>();
 
```

Comment:
This is a bad variable name: what does arrayOfAnswer mean? It is also a bad type declaration. Please avoid 'any' and user a proper type instead (e.g. AnswerOption)